vaadin-context-menu-flow-parent/vaadin-context-menu-flow/src/main/resources/META-INF/resources/frontend/contextMenuTargetConnector.js

```diff
@@ -12,6 +12,9 @@ import * as Gestures from '@vaadin/component-base/src/gestures.js';
 
     target.$contextMenuTargetConnector = {
       openOnHandler: tryCatchWrapper(function (e) {
+        if (target.preventContextMenu && target.preventContextMenu(e)) {
```

Comment:
Consider adding a comment e.g. "Used by Grid to prevent context menu on selection column click".
Otherwise it could be unclear when looking at this code where `preventContextMenu` comes from.